ding_unilogin/ding_unilogin.test

```diff
@@ -0,0 +1,160 @@
+<?php
+
```

Comment:
Technical all the new files is missing `@file` documentation.

Reply:
No: `files that contain a namespaced class/interface/trait, whose file name is the class name with a .php extension, and whose file path is closely related to the namespace (under PSR-4 or a similar standard), SHOULD NOT have a @file documentation block.` (cf. https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#file)